scripts/ci-e2e.sh

```diff
@@ -23,6 +23,7 @@ source "${REPO_ROOT}/hack/ensure-tags.sh"
 : "${OCI_ORACLE_LINUX_IMAGE_ID:?Environment variable empty or not defined.}"
 : "${OCI_UPGRADE_IMAGE_ID:?Environment variable empty or not defined.}"
 : "${OCI_ALTERNATIVE_REGION_IMAGE_ID:?Environment variable empty or not defined.}"
+: "${OCI_WINDOWS_IMAGE_ID:?Environment variable empty or not defined.}"
```

Comment:
@shyamradhakrishnan want to specifically point out this new requirement in order to run e2e tests.

Reply:
I think we should keep it optoional right?